### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by luftywiranda13**
pretty much like what I did in https://github.com/paypal/glamorous/pull/156 ✌ 😁

### CODE REVIEW

File: package-scripts.js [hunk 1]
```diff
@@ -54,13 +42,15 @@ module.exports = {
       description: 'auto-formats the code',
       script: 'prettier-eslint --write "src/**/*.js"',
     },
-    addContributor: {
```
Comment (kentcdodds) [inline package-scripts.js]: Awesome! I definitely prefer the new style. Could you update the `CONTRIBUTING.md` references to these scripts?

Comment (luftywiranda13) [inline package-scripts.js]: done!

Comment (luftywiranda13) [inline package-scripts.js]: ups, typo haha

**Comment by luftywiranda13** [comment]
yeay 🎉  
hope you don't get bored by my PR(s) 😅

**Comment by kentcdodds** [comment]
I love them! Keep them coming. Thanks!

**Pull request merged**